parser/src/snapshots/rustpython_parser__parser__tests__try.snap

```diff
@@ -167,7 +167,7 @@ expression: parse_ast
                                                         values: [
                                                             Constant(
                                                                 ExprConstant {
-                                                                    range: 114..133,
+                                                                    range: 116..123,
```

Comment:
The range of the following `FormattedValue` must be incorrect because it overlaps with this `Constant`.